<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomChat</class>
 <widget class="QWidget" name="RoomChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(48, 56, 65);</string>
  </property>
  <widget class="QListWidget" name="usersListWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>13</y>
     <width>111</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);
background-color: rgb(238, 238, 238);
font: 75 bold italic 10pt &quot;Comic Sans MS&quot;;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="chatListWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>13</y>
     <width>271</width>
     <height>231</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);
background-color: rgb(238, 238, 238);
font: 75 bold italic 10pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideLeft</enum>
   </property>
   <property name="isWrapping" stdset="0">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputTextBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>253</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);
background-color: rgb(238, 238, 238);
font: 75 bold 9pt &quot;Comic Sans MS&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);
background-color: rgb(238, 238, 238);
font: 75 bold italic 9pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="downloadButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>290</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);
background-color: rgb(238, 238, 238);
font: 75 bold italic 8pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>Download File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="uploadFileButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);
background-color: rgb(238, 238, 238);
font: 75 bold italic 9pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>Upload File</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="downloadProgressBar">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>254</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>1600000</height>
    </size>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="emojiComboBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>290</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);
background-color: rgb(238, 238, 238);
font: 75 bold 9pt &quot;Comic Sans MS&quot;;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
